### PR TITLE
Update CI configuration to add Xcode 11 and use Swift 5

### PR DESCRIPTION
**Pull request by dostrander**
- Adds xcode 11 to CI configuration
- Removes warnings in the various xcode projects used
- Updates KIF to swift 5
- Fixes gem install to not fail

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,19 @@
 language: objective-c
 matrix:
   include:
-    - osx_image: xcode9.2
+    - osx_image: xcode11.5
```
Comment (justinseanmartin) [inline .travis.yml:4]: Is 11.6 available now?

File: README.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ KIF builds and performs the tests using a standard `XCTest` testing target.  Tes
 
 **KIF uses undocumented Apple APIs.** This is true of most iOS testing frameworks, and is safe for testing purposes, but it's important that KIF does not make it into production code, as it will get your app submission denied by Apple. Follow the instructions below to ensure that KIF is configured correctly for your project.
 
+**Actively supports Xcode 11 and iOS 10-13 if you need support for an earlier version please use [v3.7.9](https://github.com/kif-framework/KIF/releases/tag/v3.7.9) or an [earlier release](https://github.com/kif-framework/KIF/releases).**
```
Comment (justinseanmartin) [inline README.md:14]: Worth including the minor version here?

File: .travis.yml (hunk 1 above)
Comment (dostrander) [reply .travis.yml:4]: Yeah i can cahnge this to 11.6

File: README.md (hunk 2 above)
Comment (dostrander) [reply README.md:14]: Will do

**Pull request merged**